docs/source/documentation.rst

```diff
@@ -695,7 +645,7 @@ GIMP software
 Training Models
 ===============
 
-To train your own model for use in AxonDeepSeg, please refer to the `IVADOMED documentation <https://ivadomed.org/tutorials/two_class_microscopy_seg_2d_unet.html>`_ on model training for two-class microscopy images.
+To train your own model for use in AxonDeepSeg, please refer to the README and accompanying code of this repository `https://github.com/axondeepseg/model_seg_generalist <https://github.com/axondeepseg/model_seg_generalist>`_.
```

Comment:
```suggestion
To train your own model for use in AxonDeepSeg, please refer to the README and accompanying code of this repository `https://github.com/axondeepseg/model_seg_generalist <https://github.com/axondeepseg/model_seg_generalist>`_.

.. NOTE :: The AxonDeepSeg development team is able to assist you with training models on your data if you do not possess the computer hardware or deep learning skills to do it yourself. When doing so, we prefer to make your newly trained model available for anyone to use with AxonDeepSeg, and if there is a susbtantual amount of work or innovation made in the process we may also ask for co-authorship from some of our lab members that contributed to this work. Please reach out to us to schedule a video-call if you'd like our assistance and to discuss more.
```

Reply:
@mathieuboudreau I agree to redirect to this repo. However if we do so I'd like to make the README prettier.